src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyle1.java

```diff
@@ -12,91 +12,81 @@
  * @author Chris Stillwell
  * @version 1.0
  */
-public class InputJavadocStyle
+
+/*
+ * Config:
+ * scope = private
```

Comment:
in your test case you have default configuration with no explicit parameters provided, so you can just put 
```
/*
 * Config: default
 */
```

Reply:
ok

Reply:
done